stdlib/multiprocessing/context.pyi

```diff
@@ -146,33 +145,43 @@ class DefaultContext(BaseContext):
 
 _default_context: DefaultContext
 
-class SpawnProcess(BaseProcess):
-    _start_method: str
-    @staticmethod
-    def _Popen(process_obj: BaseProcess) -> SpawnProcess: ...
-
-class SpawnContext(BaseContext):
-    _name: str
-    Process: ClassVar[type[SpawnProcess]]
-
 if sys.platform != "win32":
     class ForkProcess(BaseProcess):
         _start_method: str
         @staticmethod
-        def _Popen(process_obj: BaseProcess) -> Any: ...
+        def _Popen(process_obj: BaseProcess) -> popen_fork.Popen: ...
+
+    class SpawnProcess(BaseProcess):
+        _start_method: str
+        @staticmethod
+        def _Popen(process_obj: BaseProcess) -> popen_spawn_posix.Popen: ...
 
     class ForkServerProcess(BaseProcess):
         _start_method: str
         @staticmethod
-        def _Popen(process_obj: BaseProcess) -> Any: ...
+        def _Popen(process_obj: BaseProcess) -> popen_forkserver.Popen: ...
 
     class ForkContext(BaseContext):
         _name: str
         Process: ClassVar[type[ForkProcess]]
 
+    class SpawnContext(BaseContext):
+        _name: str
+        Process: ClassVar[type[SpawnProcess]]
+
     class ForkServerContext(BaseContext):
         _name: str
         Process: ClassVar[type[ForkServerProcess]]
+else:
+    class SpawnProcess(BaseProcess):
+        _start_method: str
+        @staticmethod
+        def _Popen(process_obj: BaseProcess) -> popen_spawn_win32.Popen: ...
+
+    class SpawnContext(BaseContext):
+        _name: str
+        Process: ClassVar[type[SpawnProcess]]
```

Comment:
Can we merge these class definitions with the ones in the `!= win32"` branch? Only the return type of `SpawnProcess._Popen` seems to be different; I think we can just do a conditional definition of that one staticmethod inside the class.

Reply:
But this can wait for another PR.

Reply:
Yes, I can! I'll send a new one tomorrow, any reasonable working hours are ending in my part of Russia 🙂